pkg/controller/flags/flags.go

```diff
@@ -0,0 +1,64 @@
+package flags
+
+import (
+	"flag"
+	"path/filepath"
+)
+
+const (
+	// DefaultSA is the default service account
+	DefaultSA            = "pipeline"
+	DefaultIgnorePattern = "^(openshift|kube)-"
+
+	ClusterCRName   = "cluster"
+	DefaultTargetNs = "openshift-pipelines"
+
+	// Name of the pipeline controller deployment
+	PipelineControllerName = "tekton-pipelines-controller"
+	PipelineControllerSA   = "tekton-pipelines-controller"
+
+	// Name of the pipeline webhook deployment
+	PipelineWebhookName = "tekton-pipelines-webhook"
+	SccAnnotationKey    = "operator.tekton.dev"
+)
+
+var (
+	PipelineSA      string
+	IgnorePattern   string
+	TektonVersion   = "v0.7.0"
+	ResourceWatched string
+	ResourceDir     string
+	TargetNamespace string
+	NoAutoInstall   bool
+	Recursive       bool
+)
+
+func init() {
```

Comment:
could we use a function `Parse` that gets called from `main` instead of relying on `init` magic?